src/Types/Tracks.h

```diff
@@ -20,12 +20,22 @@ class BaseTrack {
   unsigned duration;
   std::string iconUri;
   std::string addedBy;
+  bool operator==(const BaseTrack tr) {
+    return trackId == tr.trackId;
+  }
 };
 
 struct QueuedTrack : public BaseTrack {
   int votes;
   TVote currentVote;
   uint64_t insertedAt;
+  int LastPlayedxSongsAgo;
```

Comment:
whats that all about? the ranking should be based on the number of votes (`votes`) and the insertion date (`insertedAt`).
other than that, what should _last played x songs ago_ even mean? i mean, if you add a track it hasn't ever been played before (bc you just added it)

Reply:
If you add a few songs at once (same/similar insertion date) and upvote one of those it will play that one song forever in a loop. Unless i also take into account how long ago a song was last played, Which is what LastPlayedxSongsAgo is all about

Reply:
there is no last played for a song. once it has been played it is gone until someone adds it again

Reply:
same discussion here --> https://github.com/skaupper/VirtualJukebox/pull/26/files#r355657897